src/vos/evtree.c

```diff
@@ -2400,7 +2400,7 @@ evt_ssof_insert(struct evt_context *tcx, struct evt_node *nd,
 		ne->ne_child = desc_off;
 		desc = evt_off2ptr(tcx, desc_off);
 		rc = vos_dtx_register_record(evt_umm(tcx),
-					     umem_ptr2id(evt_umm(tcx), desc),
+					     umem_ptr2off(evt_umm(tcx), desc),
 					     DTX_RT_EVT, 0);
```

Comment:
ne->ne_child can be used here too or desc_off in this case

Reply:
OK, I will fix it.